pkg/fees/txFees.ts

```diff
@@ -0,0 +1,17 @@
+import fetch from "node-fetch"
+
+/**
+ * Fetch Blockstreams's API to get fee estimates for target blocks and provides 5 fee values.
+ *
+ * @returns - Array containing five fee values to choose from.
+ */
+export async function getFeesBTC(): Promise<number[]> {
+  const feeEstimates = await fetch("https://blockstream.info/api/fee-estimates").then((res) => res.json())
```

Comment:
Same here, we need to integrate the Blockstream API class and can refactor this afterwards ;)